src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -98,13 +99,24 @@ JobExecutorType(DistributedPlan *distributedPlan)
 									  "\"task-tracker\".")));
 		}
 
-		/* if we have repartition jobs with real time executor, error out */
+		/* if we have repartition jobs with real time executor and repartition
+		 * joins are not enabled, error out. Otherwise, switch to task-tracker
+		 */
 		dependedJobCount = list_length(job->dependedJobList);
-		if (dependedJobCount > 0)
+		if (dependedJobCount > 0 && !EnableRepartitionJoin)
 		{
 			ereport(ERROR, (errmsg("cannot use real time executor with repartition jobs"),
 							errhint("Set citus.task_executor_type to "
-									"\"task-tracker\".")));
+									"\"task-tracker\" or "
+									"citus.enable_repartition_joins to ON.")));
```

Comment:
Just because I'm lazy and I like copy-pasting this message. Could you add a "set" after the "or"? :)

Reply:
maybe `ON` -> `on`.